students/wachulec_maria/lesson_06_dicts_tuples_sets_args_kwargs/uppercase.py

```diff
@@ -0,0 +1,16 @@
+def capitalize(word):
+    between_large_small_distance = 32
+    word = chr(ord(word[0]) - between_large_small_distance) + word[1:]
+    return word
```

Comment:
I know it's written along with instruction, however I would personally wrote is as:
`return word[0].upper() + word[1:]`

readability on the first place